src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -142,12 +142,16 @@ object DemuxFastqs {
   def demultiplexingIterator(sources: Seq[FastqSource],
                              demultiplexer: FastqDemultiplexer,
                              threads: Int,
+                             threshold: Int,
+                             qualityEncoding: QualityEncoding,
                              batchSize: Int = DemuxBatchRecordsSize): Iterator[DemuxResult] = {
 
     require(demultiplexer.expectedNumberOfReads == sources.length,
       s"The demultiplexer expects ${demultiplexer.expectedNumberOfReads} reads but ${sources.length} FASTQ sources given.")
 
     val zippedIterator = FastqSource.zipped(sources)
+    val thresh         = qualityEncoding.toStandardAscii(PhredScore.cap(threshold + qualityEncoding.asciiOffset).toChar).toByte
```

Comment:
I think there are several things wrong with this:
1. `PhredScore.cap()` is intended to be applied to a numeric phred score, not a phred score plus some offet.  I don't think we need this call at all.
2. qualityEncoding.toStandardAscii returns the byte/char _as encoded by QualityEncoding.Standard_, which will do the wrong thing if another quality encoding is in use.

I think all this really needs to be is `(threshold + qualityEncoding.asciiOffset).to[Byte|Char].  Sorry if I led you astray, it doesn't look like there's a method on QualityEncoding to do this.